x/oracle/keeper/hooks.go

```diff
@@ -0,0 +1,32 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	marketmaptypes "github.com/skip-mev/slinky/x/marketmap/types"
+)
+
+// Hooks is a wrapper struct around Keeper.
+type Hooks struct {
+	k *Keeper
+}
+
+var _ marketmaptypes.MarketMapHooks = Hooks{}
+
+// Hooks returns registered hooks for x/oracle.
+func (k *Keeper) Hooks() Hooks {
+	return Hooks{k}
+}
+
+// AfterMarketCreated is the marketmap hook for x/oracle that is run after a market is created in
+// the marketmap.  After the market is created, a currency pair and its state are initialized in the
+// oracle module.
+func (h Hooks) AfterMarketCreated(ctx sdk.Context, ticker marketmaptypes.Ticker) error {
+	return h.k.CreateCurrencyPair(ctx, ticker.CurrencyPair)
```

Comment:
does this update everything we need on the oracle module? do we need to update the decimal points?